api/src/controllers/department-service.ts

```diff
@@ -145,7 +147,77 @@ router.get("/", passport.authenticate(["referent"], { session: false, failWithEr
   }
 });
 
-router.post("/:id/representant", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+router.get("/:cohortId/DepartmentServiceContact/export", passport.authenticate("referent", { session: false, failWithError: true }), async (req: UserRequest, res: Response) => {
```

Comment:
```suggestion
router.get("/:cohortId/DepartmentServiceContact/export", 
    accessControlMiddleware([ROLE.ADMIN, ROLE.REFERENT_DEPARTMENT, ROLE.REFERENT_REGION]), 
    requestValidatorMiddleware({ params: Joi.object({ cohortId: Joi.string().required()}) ) 
    async (req: UserRequest, res: Response) => {
```

- Pour les nouveaux endpoint on n'avait mis en place des middleware pour se simplifier les contrôles de base.
- on peux aussi typer `req` et `res` avec quelques chose comme ça :
`async (req: RouteRequest<DepartementServiceRoutes["Export"]>, res: RouteResponse<DepartementServiceRoutes["Export"]>) => {`

si tu te sens, n'hésites pas à revenir vers moi en cas de besoin.